### DISCUSSION TITLE
给appflowy的一些建议

### DISCUSSION THREAD

**Discussion by Ihopeyouarehappy** [AppFlowy-IO/AppFlowy#3729] [Ideas]
从0.2.6版本就在关注appflowy了，个人非常期待未来的1.0正式版，希望appflowy能越来越好，感谢各位的付出，加油~

我想提一些关于使用体验的建议，希望有帮助（抱歉我不会其他语言）
版本：0.3.6

1）最后一行的位置调整
现状：我在编辑文字时，页面已经滚动到最低，最后一行的位置还是太低了，感觉编辑文字时视角不方便
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/139304871/382b6ace-fe42-41f4-b504-00b9514c9141)

改善：页面滚动到最低时，下方空余出更多地方，使最后一行处于接近中间的位置
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/139304871/3bf125a5-86cf-4454-9fe1-34bccaadfcb6)

2）页面中的表格、日历、看板等区块大小的自定义控制
现状：无法改变相关区块大小
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/139304871/01cdeb88-0811-4f21-a56e-cb3dc2917d2e)

改善：增加该功能，当鼠标移动至表格区块的下方时，显示提示线（例如蓝线），拖动此线来改变表格区块的大小
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/139304871/0d9b5dcd-269f-4be1-b9e2-e077b004da9c)

3）增加拖动区块的功能（个人认为这个功能会大大提示使用体验）
现状：无法拖动区块
改善：使区块可被拖动，且可使区块与区块之间处于同一行（平行），类似notion、Anytype

4）页面滚动的改善
现状：表格、看板滚动至底部时，如果继续滚动会滚动整个页面，导致观感不适，需要来回滚动才能比较舒适的观看内容
改善：只要鼠标在区块范围内，鼠标滚轮就不会滚动页面，而只是滚动区块内的内容

5）页面滚动的改善2
现状：必须按下Shift + 鼠标滚轮才可以横向滚动内容
改善：根据不同情况滚动内容不同
当鼠标在区块的上半部分时（红区），鼠标滚轮影响区块内垂直滚动（上下滚动）。当鼠标在区块内的下半部分时（蓝区），鼠标滚轮影响区块内平行滚动（左右滚动）
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/139304871/3398d477-8f10-49cc-9c1c-49ad60f46416)

5）自定义“编辑内容”与“页面边框”的边距
现状：个人体验感觉边距太窄
改善：在设置外观中增加自定义边距的功能
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/139304871/da4b1f9a-5c0d-4bcd-aeae-72cb34a2d048)

**Comment by zhousone** [comment]
我用新浪邮箱和qq邮箱注册，都收不到注册码邮件。0.9.1版本
